client/src/search.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
For me its not clear why you have another html file.
Its just for demo? In third task you will use react router instead of creating this file?

Reply:
yes, this file is just for demo.